src/main/groovy/io/openliberty/tools/gradle/tasks/AbstractFeatureTask.groovy

```diff
@@ -38,10 +39,10 @@ public class AbstractFeatureTask extends AbstractServerTask {
 
     private InstallFeatureUtil util;
 
-    @Internal
-    Project newProject = project;
-
     private ServerFeatureUtil servUtil;
+	
+	@Internal
+ 	def jsonCoordinate;
```

Comment:
Could change `def` to a String and update the first part of the check on 134 to null and empty checks.

Reply:
I think you missed Matt's point. You would still check if it is equal, but first check that `jsonCoordinate != null and !jsonCoordinate.empty() && coordinates.equals(jsonCoordinate)`